CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/templates/FilterPluginNGTest.java

```diff
@@ -0,0 +1,299 @@
+            // intentionally left blank
+        }
+    }
+
+    /**
+     * Test of createParameters method, of class FilterPlugin.
+     */
+    @Test
+    public void testCreateParameters() {
+        System.out.println("createParameters");
+
+        final FilterPlugin instance = new FilterPluginImpl();
+        final PluginParameters result = instance.createParameters();
+
+        assertTrue(result.getParameters().containsKey(FilterPluginImpl.FILTER_TYPE_PARAMETER_ID));
+    }
+
+    /**
+     * Test of values method, of class FilterType.
+     */
+    @Test
+    public void testValues() {
+        System.out.println("values");
+
+        final FilterType[] expResult = new FilterType[3];
+        expResult[0] = FilterType.REMOVE_FILTER;
+        expResult[1] = FilterType.DESELECT_FILTER;
+        expResult[2] = FilterType.SELECT_FILTER;
+
+        final FilterType[] result = FilterType.values();
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of valueOf method, of class FilterType.
+     */
+    @Test
+    public void testValueOf() {
+        System.out.println("valueOf");
+
+        final String string = "REMOVE_FILTER";
+        final FilterType expResult = FilterType.REMOVE_FILTER;
+        final FilterType result = FilterType.valueOf(string);
+
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of getFilterTypeName method, of class FilterType.
+     */
+    @Test
+    public void testGetFilterTypeName() {
+        System.out.println("getFilterTypeName");
+
+        final FilterType instance = FilterType.REMOVE_FILTER;
+        final String expResult = "Remove";
+        final String result = instance.getFilterTypeName();
+
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of getFilterType method, of class FilterTypeParameterValue.
+     */
+    @Test
+    public void testGetFilterType() {
+        System.out.println("getFilterType");
+
+        final FilterTypeParameterValue instance = new FilterTypeParameterValue();
+        final FilterType expResult = null;
+        final FilterType result = instance.getFilterType();
+
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of validateString method, of class FilterTypeParameterValue.
+     */
+    @Test
+    public void testValidateStringWhenValid() {
+        System.out.println("validateString");
+
+        final String s = "Remove";
+        final FilterTypeParameterValue instance = new FilterTypeParameterValue();
+        final String expResult = null;
+        final String result = instance.validateString(s);
+
+        assertEquals(result, expResult);
+    }
+
+    /**
+     * Test of validateString method, of class FilterTypeParameterValue.
+     */
+    @Test
+    public void testValidateStringWhenInvalid() {
+        System.out.println("validateString");
```

Comment:
Minor thing but are you able to potentially alter the string here? Just so we're able to differentiate this test from the previous one if we need to to go through logs

Reply:
Sure thing @antares1470 I've just updated it and its ready for another review.